src/engine/strat_engine/backstore/metadata.rs

```diff
@@ -36,7 +36,54 @@ pub struct BDA {
     regions: mda::MDARegions,
 }
 
+#[derive(Debug, Clone, Copy, PartialEq)]
+enum Copy {
```

Comment:
This is using "copy" as a noun, but it could also be read as a verb, which would be confusing, because we're using it to say which version is overwritten, which is not a "copy" operation.

Anyway, how about calling this MetadataCopy or MetadataLocation so this is unambigous?

Reply:
Done, thank you.